src/styles/styles.js

```diff
@@ -386,6 +386,12 @@ const styles = {
         width: '100%',
     },
 
+    loginTermsText: {
+        color: themeColors.textSupporting,
+        fontSize: variables.fontSizeSmall,
+        lineHeight: 20,
```

Comment:
Can we try backing this down to 16 and seeing what it feels like? 

We also need to include our font family here so that we serve this up in the correct font. Just add `fontFamily: fontFamily.GTA,` and we should be good.

Reply:
Yep. Here's what it looks like on web and android. 

![image](https://user-images.githubusercontent.com/1371996/112034720-c8395100-8b0c-11eb-8ba0-fc654bb6adca.png)

![image](https://user-images.githubusercontent.com/1371996/112035230-57deff80-8b0d-11eb-8021-572ec25214b3.png)